packages/trader/src/Modules/Trading/Components/Form/TradeParams/Turbos/trade-type-tab.jsx

```diff
@@ -0,0 +1,49 @@
+import classNames from 'classnames';
+import React from 'react';
+import PropTypes from 'prop-types';
+import { ButtonToggle } from '@deriv/components';
+import { localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+
+const TradeTypeTab = ({ className, onChange }) => {
+    const [selectedValue, setSelectedValue] = React.useState('');
+    const tab_list = [
+        { text: localize('Long'), value: 'turboslong' },
+        { text: localize('Short'), value: 'turbosshort' },
+    ];
+
+    const onTabChange = e => {
+        if (e.target.value !== selectedValue.value) {
+            const name = 'contract_type';
+            const result = tab_list.find(res => res.value === e.target.value);
+            setSelectedValue(prevState => ({
```

Comment:
Why do you need an object (in `selectedValue`) if you use only `selectedValue.value`? Isn't it better to hold only `value` in state (as string, not an object)? You never use `selectedValue.name` as in `onChange` function you pass `const name = 'contract_type'`.